src/main/java/com/synopsys/integration/detect/lifecycle/run/step/RapidModeStepRunner.java

```diff
@@ -104,7 +108,12 @@ public void runOnline(BlackDuckRunData blackDuckRunData, NameVersion projectVers
         File jsonFile = operationRunner.generateRapidJsonFile(projectVersion, rapidFullResults);
         RapidScanResultSummary summary = operationRunner.logRapidReport(rapidFullResults, mode);
 
+        if (configurationFactory.componentLocationAnalysisEnabled()) {
```

Comment:
Same comment as mentioned in IntelligentModeStepRunner (just commenting for reference)